src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -787,7 +783,7 @@ export const examManagementRoute: Routes = [
         },
         data: {
             authorities: [Authority.ADMIN, Authority.INSTRUCTOR, Authority.EDITOR],
-            pageTitle: 'artemisApp.exampleSubmission.home.title',
+            pageTitle: 'artemisApp.exampleSubmission.home.LtiServicetitle',
```

Comment:
could you explain this renaming pls.

Reply:
I believe I accidentally pasted from my clipboard and didn't notice it when I committed 🤦 I've undone this. Thanks for noticing this!